igor-web/src/main/java/com/netflix/spinnaker/igor/travis/client/TravisClient.java

```diff
@@ -68,7 +68,7 @@ public abstract TriggerResponse triggerBuild(
 
   @POST("/users/sync")
   public abstract Response usersSync(
-      @Header("Authorization") String accessToken, @Body EmptyObject emptyObject);
+      @Header("Authorization") String accessToken, @Body JSONObject empty);
```

Comment:
I don't understand this change, why introduce a dependency on org.json.simple here?

Reply:
Hi, When I tested this part I was getting the error `java.lang.IllegalArgumentException: method POST must have a request body.`, I tried different workarounds, an empty String didn't work because of the JacksonConverter, the one of the JsonObject is mention [here](https://github.com/square/retrofit/issues/854#issuecomment-231111447) and there is another which uses TypedOutput, but I test again this time [EmptyObject](https://github.com/spinnaker/igor/blob/master/igor-web/src/main/java/com/netflix/spinnaker/igor/travis/client/model/EmptyObject.java) with @XmlRootElement and it's working  :O